src/entities/client/client.resolvers.js

```diff
@@ -153,6 +147,92 @@ module.exports = {
         throw e
       }
     },
+    setClientOwner: async (parent, { clientID, owner }, { models, request }) => {
+      try {
+        const { auth } = request
+        const matchingClientId = getMatchingId(clientID)
+        const [clientFromID] = await models.client
+          .get({ _id: matchingClientId })
+        const lowerCaseOwner = owner.toLowerCase()
+
+        const setOwner = async () => {
+          await models.user.get({ email: lowerCaseOwner })
```

Comment:
wenn es email heißt, dann kann man auch das hier ändern:

```suggestion
          await models.user.get({ email })
```